grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java

```diff
@@ -179,6 +177,9 @@ private Json createTask(Request request, Response response) {
                     .map(interval -> recurring(time, interval))
                     .orElse(TaskSchedule.at(time));
 
+            // Get the priority of a task (default is low)
+            priority = Optional.ofNullable(priorityParam).map(TaskState.Priority::valueOf).orElse(DEFAULT_TASK_PRIORITY);
```

Comment:
I have a test for this! You get back an IllegalArgumentException and a server 400.